en_US/deploy/cluster/auto.md

```diff
@@ -92,7 +92,7 @@ cluster {
 }
 ```
 
-### kubernetes
+### `kubernetes`
```

Comment:
Same here, I think Kubernetes is a regular word rather than code variable

Reply:
✅